Netable/Example/Models/User.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  User.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-11-28.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import Foundation
+import Netable
+
+struct User: Decodable {
+    let firstName: String
+    let lastName: String
+    let location: String
+    let bio: String
+    let age: Int
+    let loginData: LossyArray<UserLoginData>
```

Comment:
Wanted to showcase using the LossyArray type as a value within an decodable object. This is working fantastic! (The user json has been updated to contain a new parameter that will fail without lossy).